cf-testd/cf-testd.c

```diff
@@ -90,6 +104,9 @@ static const char *const HINTS[] = {
     "Enable debugging output",
     "Print the help message",
     "Print basic information about what cf-testd does",
+    "Number of jobs (threads) to run in parallel. Use '%d' in the report and"
+    " key file paths (will be replaced by the thread number, starting with 0)."
```

Comment:
maybe s/report/report filename provided by the --report option/ or something that's a bit more clear that it references the --report option value.

Reply:
Changing to "*report path and key file path*", hope that makes it clearer.